pipelines/api-pipelines.yml

```diff
@@ -136,5 +116,13 @@ stages:
               appType: 'apiApp'
               WebAppName: 'backend-$(businessUnit)-$(appName)-$(environment)-$(region)'
               packageForLinux: '$(Build.ArtifactStagingDirectory)/**/*.zip'  
-              appSettings: -CosmosSettings:ServiceEndpoint $(cosmosdbURI) -CosmosSettings:AuthKey $(cosmosdbKey) 
-                -CosmosSettings:DatabaseName linkylinkdb -CosmosSettings:ContainerName linkbundles 
+              appSettings: 
+                -CosmosSettings:ServiceEndpoint $(COSMOSDB_ENDPOINT) 
```

Comment:
I think these env variables are coming from the Pipeline infra. Probably it would be nice to add them in a comment block at the top, so new users know what to expect for configuration.

Reply:
Yes - they are coming from within the defined variable group.  Will update these in next PR for multi env deploy.